autoscale/src/main/java/com/sequenceiq/periscope/controller/DistroXAutoScaleClusterV1Controller.java

```diff
@@ -142,6 +142,13 @@ private DistroXAutoscaleClusterResponse updateClusterAutoScaleConfig(Cluster clu
             alertValidator.validateStopStartEntitlementAndDisableIfNotEntitled(cluster);
         }
         alertValidator.validateScheduleWithStopStart(cluster, autoscaleClusterRequest);
+
+        if (!cluster.getLoadAlerts().isEmpty() && autoscaleClusterRequest.getLoadAlertRequests().isEmpty()) {
```

Comment:
> Additionally. The delete*Alert methods... do those get invoked from anywhere? Are the changes required there as well ?

Yes, the changes are required there as well because those endpoints are invoked by the CLI

Reply:
> This should happen only if the modification is successful, correct? i.e. after all the changes are committed etc

This happens when a no loadAlertRequest is sent when defining autoscale policies. The presence of a pre-existing loadAlert, and no new loadAlertRequests indicates that the loadAlert is being deleted (according to the UI invocation of the backend), which will trigger STOPPED node deletion